File: ejercicios/c++/ejercicio077.cpp
#include <iostream>
using namespace std;

bool par(int);
bool impar(int);

int main() {	int numero;	cout << "ingrese un numero entero : ";	cin >> numero;
		if (impar(numero))		cout << numero << " es impar" << endl;	else		cout << numero << " es par" << endl;	return 0;}

bool par(int n){
	if(n == 0)
		return true;
	else
		return impar(n-1);
}

bool impar(int n){
	if(n == 0)
		return false;
	else
		return par(n-1);
}

File: ejercicios/python/ejercicio077.py
def par(n):
   if(n == 0):
      return True
   else:
      return impar(n-1)

def impar(n):
   if(n == 0):
      return False
   else:
      return par(n-1)

if __name__ == "__main__":
   numero = int(input("ingrese un numero : "))
   if (impar(numero)):
      print(str(numero) + " es impar")
   else:
      print(str(numero) + " es par")